src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -58,7 +58,7 @@ export default [
 
             const isAttachment = _.has(reportAction, 'isAttachment')
                 ? reportAction.isAttachment
-                : ReportUtils.isReportMessageAttachment(text);
+                : ReportUtils.isReportMessageAttachment(message);
```

Comment:
Are you sure that message will always be a Object here?

Reply:
I have tested in different scenarios and it does return object. But One improvement that can be made is:
```diff
-            const message = _.last(lodashGet(reportAction, 'message', null));
+            const message = _.last(lodashGet(reportAction, 'message', [{}]));

```

Reply:
Yup. Good catch. lets do this.